tests/test_linked_list.py

```diff
@@ -11,25 +11,38 @@ def test_block_insert():
     block = Block()
 
     assert block.is_empty
+
     assert block.first_op is None
     assert block.last_op is None
+
     assert list(block.ops) == []
 
     block.add_op(a)
 
-    assert not block.is_empty
```

Comment:
Not sure why this assert was removed?
Even though it's obvious since we check that `first_op` and `last_op` are not `None`, we should still check it?

Reply:
that's a merge accident, I'll add it back in